src/librustc/diagnostics.rs

```diff
@@ -168,6 +168,11 @@ match x {
 ```
 "##,
 
+E0010: r##"
+The value of static and const variables must be known at compile time, and they
```

Comment:
I think using 'constants' is worse, since `static`s are not constants (either in theory or in practice).